src/utils/default.nix

```diff
@@ -125,6 +125,7 @@ in
         ${coreutils}/bin/rm -rf $TMPDIR
       '';
 
+    # TODO is this really needed? Seems to make builds slower, why not unpack + build?
```

Comment:
I changed a bunch of runCommands to runCommandLocal so that at least it wouldn't send unpacked archives back and forth to the binary cache. Saves a whole bunch of lookups on large projects, too.